slides/boosting/references.bib

```diff
@@ -0,0 +1,58 @@
+@article{kneib2009variable,
+  title={Variable selection and model choice in geoadditive regression models},
+  author={Kneib, Thomas and Hothorn, Torsten and Tutz, Gerhard},
+  journal={Biometrics},
+  volume={65},
+  number={2},
+  pages={626--634},
+  year={2009},
+  publisher={Oxford University Press},
+  url={https://epub.ub.uni-muenchen.de/2063/1/tr003.pdf}
+}
+
+@article{hofner2014model,
+  title={Model-based boosting in R: a hands-on tutorial using the R package mboost},
+  author={Hofner, Benjamin and Mayr, Andreas and Robinzonov, Nikolay and Schmid, Matthias},
+  journal={Computational statistics},
+  volume={29},
+  pages={3--35},
+  year={2014},
+  publisher={Springer},
+  url={https://cran.r-project.org/web/packages/mboost/vignettes/mboost_tutorial.pdf}
+}
+
+@misc{kumar2019kaggle,
+  author = {Kumar, Ajay},
+  title = {Life Expectancy (WHO)},
+  year = {2019},
+  publisher = {Kaggle},
+  url={https://www.kaggle.com/datasets/kumarajarshi/life-expectancy-who}
+}
+
+@article{kearnsour,
+  title={Our goals are to offer some simple observations and results that seem related to the above questions, with the eventual goal of resolving the Hypothesis Boosting Problem. We begin with the definitions of strong and weak learnability. Let and?` be parameterized classes of representations of Boolean functions; that is,=},
+  author={Kearns, Michael},
+  url={https://www.cis.upenn.edu/~mkearns/papers/boostnote.pdf}
+}
+
+@article{schapire1990strength,
+  title={The strength of weak learnability},
+  author={Schapire, Robert E},
+  journal={Machine learning},
+  volume={5},
+  pages={197--227},
+  year={1990},
+  publisher={Springer},
+  url={https://link.springer.com/content/pdf/10.1007/BF00116037.pdf}
+}
+
+@inproceedings{freund1996experiments,
+  title={Experiments with a new boosting algorithm},
+  author={Freund, Yoav and Schapire, Robert E and others},
+  booktitle={icml},
+  volume={96},
+  pages={148--156},
+  year={1996},
+  organization={Citeseer},
+  url={https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=d186abec952c4348870a73640bf849af9727f5a4}
+}
```

Comment:
For everything to work smoothly with citelink in the slide titles, we unfortunately need all bib keys in caps for all references.bib files, i.e., freund1996experiments => FREUND1996EXPERIMENTS